.github/CODEOWNERS

```diff
@@ -321,3 +321,5 @@
 /modules/xresources.nix                               @rycee
 
 /modules/xsession.nix                                 @rycee
+
+/modules/xmobar.nix                                   @t4ccer
```

Comment:
This should be `/modules/programs/xmobar.nix`, and you should add `/tests/modules/programs/xmobar`.

The list is ordered alphabetically, but you should group the tests line just after the module line.